aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -274,6 +274,10 @@ TORCH_META_FUNC(amax)
             self.scalar_type(), " for input's dtype and ",  maybe_result.scalar_type(), " for out's dtype.");
   }
   if (self.numel() == 0) {
+    TORCH_CHECK(
```

Comment:
Should this be in `zero_numel_check_dims`?

Reply:
Hi, @kshitij12345!
Yes, thanks for pointing it out :) I've moved it.